### DISCUSSION TITLE
Feature Requests and Feedbacks

### DISCUSSION THREAD

**Discussion by rohankishore** [rohankishore/Aura-Text#7] [Q&A]
This discussion page is to collect feedback and future ideas from you guys!

**Comment by rohankishore** [comment]
Ok 'Tabs' is the #1 request i got from most of the users... Bringing tabs to a tkinter app is a hard job but i'm trying my level best.. I'll try to bring the feature in the next update... Your patience is much appreciated! Thanks

**Comment by rohankishore** [comment]
@matthewyang204 im implementing this function grid kinda thingy. do test it and tell me how's it. `Ctrl+Shift+W` to trigger it, use arrows to navigate.

also i've implemented keymap configs and all.

**Reply by matthewyang204** [reply]
It seems great. It works okay in most cases, though some things aren't implemented yet and will inevitably cause the program to crash when I trigger them. Also, I think that the splash screen should be enabled by default in the program's copy of LocalAppData.

**Reply by rohankishore** [reply]
Users can enable it in the onboarding menu.

**Reply by matthewyang204** [reply]
It is a good idea to enable it by default anyways though. I created a PR #144 for it.

**Reply by rohankishore** [reply]
I've merged it! Thanks. 

Also, post any feature ideas here, no matter how crazy.

**Comment by Om-Mochi** [comment]
For Intelligence like Auto Completion, Hover, Signature, Goto Defination etc like features you need to implement LSP servers like for python Pylsp, for cpp/c clangd etc. See my code editor image i already done it. Already VS Code,JetBrains IDE are uses diffrent servers for this features.
<img width="1920" height="1039" alt="GruwBocClassic" src="https://github.com/user-attachments/assets/c581d60f-d78b-457f-a5ee-ba4a5874005f" />
